Makefile

```diff
@@ -37,23 +37,65 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-# Build actions
-all: manager injector handler
+# we define target specific variables values https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html
+injector handler: BINARY_PATH=./cli/$(BINARY_NAME)
+manager: BINARY_PATH=main.go
+
+docker-build-injector: IMAGE_TAG=$(INJECTOR_IMAGE)
+docker-build-handler: IMAGE_TAG=$(HANDLER_IMAGE)
+docker-build-manager: IMAGE_TAG=$(MANAGER_IMAGE)
+
+docker-build-ebpf:
+	docker build --platform linux/$(OS_ARCH) --build-arg ARCH=$(OS_ARCH) -t ebpf-builder-$(OS_ARCH) -f bin/ebpf-builder/Dockerfile ./bin/ebpf-builder/
+	-rm -r bin/injector/ebpf/
+	docker run --rm -v $(shell pwd):/app ebpf-builder-$(OS_ARCH)
+
+lima-push-injector lima-push-handler lima-push-manager: FAKE_FOR=COMPLETION
```

Comment:
This aims to provide bashcompletion or similar as they are parsing the Makefile, not necessarily the templates